target/linux/rockchip/patches-5.10/202-net-stmmac-Use-phy_find_first-for-invalid-phy_addr.patch

```diff
@@ -0,0 +1,32 @@
+From d6a39612929727373b0c3e1a42839f0cac96da86 Mon Sep 17 00:00:00 2001
+From: hmz007 <hmz007@gmail.com>
+Date: Mon, 5 Jul 2021 14:47:15 +0800
+Subject: [PATCH 3/4] net: stmmac: Use phy_find_first() for invalid phy_addr
```

Comment:
Why not use the correct PHY address in the first place? The datasheet for the PHY is available.

Reply:
According to the datasheet here: https://datasheet.lcsc.com/szlcsc/2106070236_Motorcomm-YT8521SH-CA_C2685354.pdf

**5.2. PHY Address** 

> For YT8521S, Strapping PHYAD[2:0] is used to generate phy address. YT8521S always responses to phy address 0. It can be disabled by configure bit[6] to 1'b0 of extended register(address 0xa005). It also has another broadcast phy address which is configurable through mdio. Bit[4:0] of extended register(address 0xa005) is broadcast phy address and its default value is 5’b11111. Bit[5] of extended register(address 0xa005) is enable control for broadcast phy address and its default value is 1’b0


My reading of that paragraph is that PHY address is writable but doesn't have a default value?